Dockerfile

```diff
@@ -1,25 +1,28 @@
-# Stage 1 - the build react app
-FROM node:12.4.0-alpine as build-deps
+# Stage 1 - install the heavy dependencies
+FROM --platform=$BUILDPLATFORM node:lts-alpine as build-deps
 WORKDIR /usr/src/app
-COPY client/package.json client/package-lock.json ./
-RUN npm i
+COPY client/package.json ./
+RUN npm install --quiet
```

Comment:
What are your thoughts on using `--quiet` here?

Reply:
I used the quiet flag after hitting the travis console limit for FOSS projects to reduce the overall console output for the Multiarch-Build.